scripts/pod.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
I think we could either rename this to make it clearer that it's an example, or move this inline to something like a scripts/README.md (if it doesn't already exist)?

Reply:
I updated this file's name to be `sync_pod.yaml` in the other PR. A README.md sounds like a good addition though so that these scripts could be understood better by future contributors though.